docs/Model/GetTransacEmailsListTransactionalEmails.md

```diff
@@ -9,7 +9,9 @@ Name | Type | Description | Notes
 **messageId** | **string** | Message Id of the sent email | 
 **uuid** | **string** | Unique id of the email sent to a particular contact | 
 **date** | [**\DateTime**](\DateTime.md) | Date on which transactional email was sent | 
```

Comment:
Please change it to `[**\DateTime**]`
as mentioned in https://docs.google.com/document/d/169kfKuUCVAF2PSkdfAazUY5G-7EL6DDLR_XRihe2pWw/edit#

Reply:
I've made the changes.